packages/nextjs/src/edge/types.ts

```diff
@@ -0,0 +1,5 @@
+// We cannot make any assumptions about what users define as their handler except maybe that it is a function
+export interface EdgeRouteHandler {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (req: any): any | Promise<any>;
```

Comment:
l: Should we use `unknown`?

Reply:
Unfortunately, we have to go with any here because making it unknown would cause TS errors when people pass in their original functions into the wrappers.